CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Changed
+- BoTorch dependency bumped to `>=0.9.3`
```

Comment:
Are there any major differences compared to the old PR implementing this that I should be aware of?

Reply:
no
I remade the PR because even merge was difficult because some entire file locations changed